src/libraries/System.Private.CoreLib/src/System/Globalization/TimeSpanParse.cs

```diff
@@ -563,20 +564,22 @@ internal bool SetBadFormatSpecifierFailure(char? formatSpecifierCharacter = null
             }
         }
 
-        internal static long Pow10(int pow)
+        internal static int Pow10(int pow)
```

Comment:
>Pow10 [](http://example.com/codeflow?start=28&length=5)

Now, as this method is limited to to exponent 7, I suggest rename it to something like `Pow10UpToExponent7` to avoid anyone in the future call it in wrong way.

Reply:
Changed the name to `Pow10UpToMaxFractionDigits`

Reply:
We could remove the method and just index into a `FractionDigitsToPow10Table` property. This might allow elimination of bounds checks at call sites.